packages/dcos-integration-test/extra/test_3dt.py

```diff
@@ -688,3 +688,25 @@ def test_3dt_runner_poststart(dcos_api_session):
     }
 
     dcos_api_session.metronome_one_off(poststart_job)
+
+
+def test_3dt_runner_cluster(dcos_api_session):
```

Comment:
It's to make sure `3dt check cluster` succeeds since the installer now runs it during postflight.